src/editmap.cpp

```diff
@@ -222,7 +222,7 @@ void editmap_hilight::draw( editmap &em, bool update )
                 char t_sym = terrain.symbol();
                 nc_color t_col = terrain.color();
 
-                if( g->m.furn( p ) > 0 ) {
+                if( g->m.furn( p ).to_i() > 0 ) {
```

Comment:
We really need a `is_null` function in `int_id`.

Reply:
I fully agree, but that should perhaps come in a separate PR.